### PR TITLE
Tests documentation

### PR DESCRIPTION
**Pull request by pablisco**
Adds documentation and badges on project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,20 +23,57 @@ In your `build.gradle` file, add this line:
...
-## Testing
+## Usage
+
+The main entity used to access to platform services is [Instance](pusher-platform-core/src/main/kotlin/com/pusher/platform/Instance.kt).
+
+The simpler way to create a new instance uses 4 things:
 
-### TODO
+* locator: Instance locator id in the form of 'v1:us1:1a234-123a-1234-12a3-1234123aa12'
+* serviceName: the name of the service this instance will use
```
Comment (hamchapman) [inline README.md]: Nitpick: `the` should have a capital in `the name`

File: README.md [hunk 2]
```diff
@@ -21,20 +23,57 @@ In your `build.gradle` file, add this line:
...
+        new AndroidDependencies()
+    );
+}
+
+```
+
+
+## Testing
 
+We use Junit 5 and/or Spek to run unit and integration tests. In `pusher-platform-core` we have two test folders. Unit tests can be located in `test` and integration tests in `integrationTest`. In order to run integration tests locally have to do a couple of extra steps. They can be found [here](pusher-platform-core/src/integrationTest/Readme.md).
```
Comment (hamchapman) [inline README.md]: > run integration tests locally have

should be

> run integration tests locally we have

File: pusher-platform-core/src/integrationTest/Readme.md [hunk 3]
```diff
@@ -10,7 +10,4 @@ The tester can be found here: https://github.com/pusher/platform-sdk-tester (Pri
 running it locally there is nothing else to set up. If you have a remote server when you have to 
 provide the following values in your `~/.gradle.properties` or as `envvars`:
 
- - sdk_tester_url
- - sdk_tester_username (May be optional)
- - sdk_tester_password (May be optional)
-
+ - sdk_tester_url: The address to where the sdk tester is located (`localhost:10443` by default)
```
Comment (hamchapman) [inline pusher-platform-core/src/integrationTest/Readme.md]: Is it a full URL or just a host?

File: README.md [hunk 4]
```diff
@@ -21,20 +23,57 @@ In your `build.gradle` file, add this line:
...
 
-## Testing
+## Usage
+
+The main entity used to access to platform services is [Instance](pusher-platform-core/src/main/kotlin/com/pusher/platform/Instance.kt).
+
+The simpler way to create a new instance uses 4 things:
 
-### TODO
+* locator: Instance locator id in the form of 'v1:us1:1a234-123a-1234-12a3-1234123aa12'
```
Comment (hamchapman) [inline README.md]: Nitpick: I'd put locator, serviceName etc in backticks to get `locator`, `serviceName` etc

File: pusher-platform-core/src/integrationTest/Readme.md (hunk 3 above)
Comment (pablisco) [reply pusher-platform-core/src/integrationTest/Readme.md]: Just the host.
Changed it to:
```
 The host (without schema) of where the SDKtester is located (`localhost:10443` by default)
```

**Pull request merged**